For native Windows, the PKG_CONFIG_PATH separator is ; rather than :.
---
 check/check-sort-order | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/check/check-sort-order b/check/check-sort-order
index 77c0806..6482f6f 100755
--- a/check/check-sort-order
+++ b/check/check-sort-order
@@ -34,8 +34,9 @@ set -e
 
 . ${srcdir}/common
 
-order1="$srcdir/sort/sort:$srcdir/sort"
-order2="$srcdir/sort:$srcdir/sort/sort"
+[ "x$native_win32" = "xyes" ] && sep=';' || sep=':'
+order1="${srcdir}/sort/sort${sep}${srcdir}/sort"
+order2="${srcdir}/sort${sep}${srcdir}/sort/sort"
 
 export PKG_CONFIG_PATH
 PKG_CONFIG_PATH="$order1"
-- 1.8.1.4 